caddytest/caddytest.go

```diff
@@ -272,7 +273,7 @@ func CreateTestingTransport() *http.Transport {
 		IdleConnTimeout:       90 * time.Second,
 		TLSHandshakeTimeout:   5 * time.Second,
 		ExpectContinueTimeout: 1 * time.Second,
-		TLSClientConfig:       &tls.Config{InsecureSkipVerify: true},
+		TLSClientConfig:       &tls.Config{InsecureSkipVerify: true}, //nolint:gosec
```

Comment:
Do you think it's better to have this comment or add this exclusion rule to the `.golangci.yml`?
```yml
    - path: caddytest/caddytest.go
      text: "G402" # G402: TLS InsecureSkipVerify set true. (gosec)
      linters:
        - gosec
```

I'm conflicted myself

Reply:
Personally I prefer the inline `//nolint` comments, because they work in all cases, and don't depend on the presence of one particular config. They also communicate that there's something fishy going on here, and we accept the risk 😉